packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -311,125 +334,139 @@ const App = (props) => {
                     defaultLocale={DEFAULT_LOCALE}
                 >
                     <CurrencyProvider currency={currency}>
-                        <Seo>
-                            <meta name="theme-color" content={THEME_COLOR} />
-                            <meta name="apple-mobile-web-app-title" content={DEFAULT_SITE_TITLE} />
-                            <link
-                                rel="apple-touch-icon"
-                                href={getAssetUrl('static/img/global/apple-touch-icon.png')}
-                            />
-                            <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
-
-                            {/* Urls for all localized versions of this page (including current page)
-                            For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
-                            {site.l10n?.supportedLocales.map((locale) => (
+                        <StoreLocatorParamsProvider>
```

Comment:
What's the difference between StoreLocatorParamProvider and StoreLocatorProvider? What params are stored in the SLPP that we can't access from the URL? In another word, do we really need an additional provider here?